cypress/integration/features/course/searchRoom.feature

```diff
@@ -0,0 +1,27 @@
+Feature: TO search for a course via serach input box
+
+  As a user (teacher & student) I want to search for dedicated course so that I can quickly find it.
+
+  Scenario: teacher is able to search for a course and find it
+    Given I am logged in as a 'teacher' at 'brb'
+    When I go to rooms overview
+    When I enter the course name 'Biologie' into the search field
+    Then I see the course 'Biologie' on the room overview page
+
+  Scenario: teacher is able to search for a course and DO NOT find it
+    Given I am logged in as a 'teacher' at 'brb'
+    When I go to rooms overview
+    When I enter the course name 'Cy::NotExistingMathematikCourse' into the search field
+    Then I do not see the course 'Cy::NotExistingMathematikCourse' on the room overview page
+
+  Scenario: student is able to search for a course and find it
+    Given I am logged in as a 'student' at 'brb'
+    When I go to rooms overview
+    When I enter the course name 'Course with subject and tasks' into the search field
+    Then I see the course 'Course with subject and tasks' on the room overview page
```

Comment:
* Is it enough if the student enters a part of the course name, for example "subject and tasks"? Then also the course should be available in the results.

Reply:
yes also works that way, updated the test.